dspace-api/src/main/java/org/dspace/app/requestitem/RequestItemEmailNotifier.java

```diff
@@ -188,34 +197,48 @@ public void sendResponse(Context context, RequestItem ri, String subject,
         // Attach bitstreams.
         try {
             if (ri.isAccept_request()) {
-                if (ri.isAllfiles()) {
-                    Item item = ri.getItem();
-                    List<Bundle> bundles = item.getBundles("ORIGINAL");
-                    for (Bundle bundle : bundles) {
-                        List<Bitstream> bitstreams = bundle.getBitstreams();
-                        for (Bitstream bitstream : bitstreams) {
-                            if (!bitstream.getFormat(context).isInternal() &&
-                                    requestItemService.isRestricted(context,
-                                    bitstream)) {
-                                // #8636 Anyone receiving the email can respond to the
-                                // request without authenticating into DSpace
-                                context.turnOffAuthorisationSystem();
-                                email.addAttachment(
-                                        bitstreamService.retrieve(context, bitstream),
-                                        bitstream.getName(),
-                                        bitstream.getFormat(context).getMIMEType());
-                                context.restoreAuthSystemState();
+                if (ri.getAccess_token() != null) {
+                    // {6} secure access link
+                    email.addArgument(configurationService.getProperty("dspace.ui.url")
+                            + "/items/" + ri.getItem().getID()
+                            + "/access-by-token?accessToken=" + ri.getAccess_token());
```

Comment:
This doesn't match the latest path from https://github.com/DSpace/dspace-angular/pull/3984 anymore, I think you can remove `/access-by-token`

Reply:
@ybnd I've pushed a change that accepts either the +7DAYS string delta format, or a valid date string, and uses that (and if delta, the decision date) to store an accessExpiry field as timestamp in the database.

(note this means the sql migration script has changed)